cdap-spark-core3_2.12/src/k8s/Dockerfile

```diff
@@ -43,6 +43,18 @@ USER root
 RUN apt-get update && \
   apt-get -y install curl && \
   curl -L -o /opt/cdap/cdap-spark-core/lib/gcs-connector-hadoop2-latest.jar https://storage.googleapis.com/hadoop-lib/gcs/gcs-connector-hadoop2-latest.jar && \
+  # start aws dependency
+  curl -L -o /opt/cdap/cdap-spark-core/lib/hadoop-aws-2.9.2.jar https://repo1.maven.org/maven2/org/apache/hadoop/hadoop-aws/2.9.2/hadoop-aws-2.9.2.jar && \
+  curl -L -o /opt/cdap/cdap-spark-core/lib/aws-java-sdk-bundle-1.11.199.jar https://repo1.maven.org/maven2/com/amazonaws/aws-java-sdk-bundle/1.11.199/aws-java-sdk-bundle-1.11.199.jar && \
+  curl -L -o /opt/cdap/cdap-spark-core/lib/jackson-dataformat-cbor-2.8.8.jar  https://repo1.maven.org/maven2/com/fasterxml/jackson/dataformat/jackson-dataformat-cbor/2.8.8/jackson-dataformat-cbor-2.8.8.jar && \
```

Comment:
why do we need all of these? shouldn't the bundle contain everything needed?

Reply:
The bundle relocated the third-party libraries, they’re no longer available to use under their original import names according to https://aws.amazon.com/blogs/developer/java-sdk-bundle/. I verified without some of these dependencies, the pipeline will fail with some class not found exception

Reply:
many of these are explicitly excluded at https://github.com/cdapio/cdap/blob/426a85da838b11c9f20a4ce5bc3f13270020998a/cdap-spark-core3_2.12/pom.xml#L397. I would be surprised if this works properly.

Even if it does work, it would be better to handle it in the pom rather than adding things here.

Reply:
Which classes were not found when you didn't include these dependencies?

Reply:
So initially it failed with missing http class in `net.java.dev.jets3t` dependency, that got fixed by pulling in, this dependency is only needed when s3n is used. (https://hadoop.apache.org/docs/stable/hadoop-aws/tools/hadoop-aws/s3n.html) 
Later it failed with missing `StringUtils` in commons-lang, I think these are the only two extra dependencies that are required. Removed others to keep the dependency clean, added them initially just to make sure they are same as sandbox